projects/ci-cd.md

```diff
@@ -0,0 +1,130 @@
+| GitHub         | [Run with Telemetry GitHub Action](https://github.com/krzko/run-with-telemetry)                                                                             |                                                                                                    | * Traces via GitHub action                                    |
+| GitLab         | [Distributed Tracing](https://docs.gitlab.com/ee/operations/tracing.html)                                                                                   |                                                                                                    | * Tracing in GitLab pipelines                                 |
+| pytest         | [pyTest OpenTelemetry plugin](https://pypi.org/project/pytest-otel/)                                                                                        |                                                                                                    | * Tracing of Python test                                      |
+| Maven          | [Maven OpenTelemetry extension](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/maven-extension/README.md)                           |                                                                                                    | * Tracing of Maven builds                                     |
+| Ansible        | [Ansible OpenTelemetry Callback Plugin](https://docs.ansible.com/ansible/latest/collections/community/general/opentelemetry_callback.html)                  |                                                                                                    | * Tracing in Ansible playbooks                                |
+| JUnit          | [JUnit to OpenTelemetry](https://github.com/mdelapenya/junit2otlp)                                                                                          |                                                                                                    | * Tracing of JUnit test results                               |
+| Eiffel         | [Eiffel CI/CD Event Protocol](https://eiffel-community.github.io/)                                                                                          |                                                                                                    | * Event protocol for CI/CD observability and interoperability |
+| CDEvents      | [CD Event Protocol](https://cdevents.dev/)                                                                                                                  |                                                                                                    | * Event protocol for CI/CD observability and interoperability |
```

Comment:
Why does c7cca2b712f2eabb37f8dcd9fa8ddc4415a9ec7f partially revert c3cdd07cb183a75e3e82847812a85107ea765c8b?
It removes Tekton from this table.

Reply:
Thanks for catching that. It was unintentional. Just pushed the fix.